### DISCUSSION TITLE
Releases made by a specific account are not appearing

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [community/community#101667] [Repositories]
### Select Topic Area

Bug

### Body

I don't know if this should be in repository or API, but...

Releases made by @LizardByte-bot are hidden in all my repositories.

I can manually edit a release, and then it will re-appear, but all others are missing.

See here: https://github.com/LizardByte/Sunshine/releases
Same results in the api: https://api.github.com/repos/LizardByte/Sunshine/releases

But I can manually goto a release page, and it will exist.

https://github.com/LizardByte/Sunshine/releases/tag/v0.19.0
https://api.github.com/repos/LizardByte/Sunshine/releases/tags/v0.19.0

Releases made by other accounts (my regular account, or github action bot) appear as they should.

I checked everything I could related to that account, and I don't see anything obvious.

- Has the correct permission in my org.
- 2FA is enabled.
- Nothing suspicious in security log.

I renamed the account and changed the email a long time ago, I don't know if that is part of the cause. But the issue just started in the last 12 hours.

**Comment by spenserblack** [comment]
Wow, I just went down a rabbit-hole and I'm completely stumped. At first my initial assumption was that the "hidden" releases were just drafts, but it quickly became obvious that they weren't drafts. Eventually I did a side-by-side diff of the API response from v0.19.0 (hidden) and v0.21.0 (visible), but even then I still didn't find anything significant. The [release's tags](https://github.com/LizardByte/Sunshine/tags) already have notes, so there's nothing missing there, either.

**Reply by ReenigneArcher** [reply]
I'm at a loss too. They were all up yesterday. Even drafts should appear in the `/releases` endpoint.

I put a ticket in with GitHub as well and they closed it with a generic response that they don't release inactive usernames... they obviously didn't read my ticket.

I was hoping it was related to the incident earlier today, but it doesn't seem to be the case.

I think I'll write an automation to loop over each repo in my org, and each release, and perform an "edit" with no changes. Hopefully that will be enough to bring them back. Not really how I wanted to spend my evening.

**Reply by ReenigneArcher** [reply]
Actually, I'll have to loop over each tag... since they don't appear in the releases endpoint, ha.

**Reply by ReenigneArcher** [reply]
@spenserblack so... apparently my bot account was flagged and is in kind of a hidden state.

I discovered after I wrote the automation to edit each release and found that it could only make 60 api requests per hour instead of the normal 5000.

I'm assuming that someone got upset after my bot commented on a stale issue and they flagged it as spam or something. What irritates me is that I got no email or notification from GitHub that the account was going to be flagged or suspended, and even when I log in with the account there is nothing obvious that it's suspended. I only discovered it was suspended when I went to contact support and only saw 3 options instead of the normal amount of choices... I'm also seriously doubting that GitHub did any investigation to the claims of the report and just immediately flagged.

**Reply by spenserblack** [reply]
Strange. I really don't know what GitHub's doing, but maybe it's "hidden" because the investigation is ongoing? If it was found to actually be spam, I'd assume that GitHub would flat out ban it so it couldn't even make releases (or perform other actions) in the first place.

Depending on how you work, you might be able to use workflows/actions to create releases instead of using a bot. Personally, I usually do one of these two:
1. Parse out the contents of a tag's message to be the release title and body.
2. Utilize GitHub's automatic release notes. This really relies on keeping your PRs clean, though, so no "tons of updates" PRs :sweat_smile:

**Reply by ReenigneArcher** [reply]
I use a lot of workflows and automations. I typically use a PAT as I wanted the account to represent my GitHub org. Also, I run a workflow after a release is made... for this to work, you must use a PAT, as the github actions bot does not trigger other workflows.

I also have workflows that work between multiple repositories, for which the default github actions bot does not have permission to do.

**Reply by spenserblack** [reply]
:thinking: Yeah, with those requirements it's pretty much unavoidable to use a bot, or at the very least a dedicated account's PAT. I'm curious if your bot will eventually get un-banned (:pray:).

**Reply by ReenigneArcher** [reply]
They have unblocked the account!

**Comment by K-O-V-I-D** [comment]
I have created two support tickets, it's has been 15+ days with no response from the support team, actually my account was flagged and I don't even know the reason!! Can anyone let me know where  I can get a sooner reply as my campus placements are starting..please help!!
